instances/latest/commonCoordinateSpaceVersions/MBISC/MBISC_v2012.jsonld

```diff
@@ -0,0 +1,125 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.om-i.org/props/"
+    },
+    "@id": "https://openminds.om-i.org/instances/commonCoordinateSpaceVersion/MBISC_v2012",
+    "@type": "https://openminds.om-i.org/types/CommonCoordinateSpaceVersion",
+    "abbreviation": "MBISC_v2012",
+    "author": null,
+    "custodian": null,
+    "description": "'The Marmoset Brain in Stereotaxic Coordinates' (2012), page IX, Figure B and \"Histology\" section explains: The horizontal zero plane passes through the lower margin of the orbit and the center of the external auditory meatus. The anteroposterior zero plane is perpendicular to the horizontal zero plane and passes through the centers of the external auditory meatus. The left-right zero plane is the midsagittal plane.",
+    "digitalIdentifier": {
+        "@id": "https://openminds.om-i.org/instances/ISBN/978-0-12-415818-4"
+    },
```

Comment:
This most likely will fail. I will get back to you about the correct way of writing typically non-resolvable PIDs (ISBN, ISSN, etc) within openMINDS.